nautobot/docs/configuration/required-settings.md

```diff
@@ -89,6 +86,9 @@ Caching is configured by defining the [`CACHEOPS_REDIS`](#cacheops_redis) settin
 
 For more details Nautobot's caching see the guide on [Caching](../../additional-features/caching).
```

Comment:
Are any changes needed in the `additional-features/caching.md` document (which I admit I've never read before)?

Reply:
I did add a small admonition there, but other than that I don't think anything else needs to be added.